api/python/quilt3/main.py

```diff
@@ -101,6 +100,8 @@ def _launch_local_s3proxy():
     Launches an s3 proxy (via docker)
     on localhost:5002
     """
+    import dns.resolver
```

Comment:
I didn't add `dnspython` as dependency because this function is actually unused.
https://github.com/quiltdata/quilt/blob/fea619d385a87321c53e164c38fbb953c509a0e1/api/python/quilt3/main.py#L110-L112

It might be a good time to remove this function.

Reply:
@kevinemoore can we safely remove `_launch_local_s3proxy()`?

Reply:
Instead of removing it, I'd rather see us use it to avoid sending local-mode traffic to open.quiltdata.com. I don't believed it's being called currently so we could remove it. Maybe it's safest to remove it temporarily and bring that functionality back when we fix local-mode packages.